### PR TITLE
feat(player): add updateStats on player to change bot activity

### PR DESCRIPTION
**Pull request by Psykka**

### CODE REVIEW

File: src/listeners/lavalink/player/playerCreate.js [hunk 1]
```diff
@@ -10,10 +9,7 @@ export default class extends Listener {
   }
 
   run (player) {
-    const size = this.client.music.players.size
-    const activity = `${size} player${size === 1 ? '' : 's'}`
-
-    this.client.user.setActivity(activity, { type: ActivityType.Listening })
+    player.manager.updateStats()
```
Comment (miguelmoraesp) [inline src/listeners/lavalink/player/playerCreate.js:12]: could be only `this.updateStats()` because Listener class already extends the entire client

**Review by miguelmoraesp — DISMISSED** [review]

File: src/listeners/lavalink/player/playerCreate.js (hunk 1 above)
Comment (Psykka) [reply src/listeners/lavalink/player/playerCreate.js:12]: I'll figure it out

Comment (Psykka) [reply src/listeners/lavalink/player/playerCreate.js:12]: `this` is a listener class no a player class

**Review by miguelmoraesp — APPROVED** [review]
LGTM

**Pull request merged**